### ISSUE TITLE
Creating in different namespace

### ISSUE THREAD

**Issue by shinji62** [jenkinsci/kubernetes-operator#77]
Hi,
I try to create a CRD in a different namespace from the operator, but I can't.

Kubectl says 
```
kubectl apply -f my-jenkins/jenkins-vif-team/deployment/jenkins.yml -n vif-team-ns                                                                                           
jenkins.jenkins.io/jenkins-vif configured
```

But in the operator logs just nothing happen...
```
h of *v1.Event ended with: The resourceVersion for the provided watch is too old.
W0821 09:02:53.920845       1 reflector.go:270] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:196: watch of *v1.Event ended with: The resourceVersion for the provided watch is too old.
W0821 09:12:07.996012       1 reflector.go:270] sigs.k8s.io/controller-runtime/pkg/cache/internal/informers_map.go:196: watch of *v1.Event ended with: The resourceVersion for the provided watch is too old.


then nothing...



```

Not sure how to fix that, I use the latest on the master branch and kubernetes is 1.12

**Comment by tomaszsek** [comment]
Hi @shinji62,

Please follow the instructions https://github.com/jenkinsci/kubernetes-operator/blob/master/documentation/security.md#setup-jenkins-operator-and-jenkins-in-separated-namespaces.

Cheers

**Comment by shinji62** [comment]
Hi,

I use `WATCH_NAMESPACE=""` which from the operator sdk mean check all namespace.
I have created  `ClusterRole/Binding`  for the operator which works well. M

I can create a CRD in a different namespace but it seems the operator try to reach the Jenkins master using  the service name `jenkins-operator-http-jenkins-vif`, without the namespace like `jenkins-operator-http-jenkins-vif.vif-team-ns.svc.cluster.local`

So CRD is working but the operator can NOT manage it.

**Comment by tomaszsek** [comment]
Hi @shinji62,

Yes, you are right it is already fixed in `v0.2.0` which will be released soon. In the meantime you can use version from master but you have to build it and host by yourself, sorry for the inconvenience.

Cheers

**Comment by shinji62** [comment]
Ok, so I will wait 0.2.0
